build.gradle

```diff
@@ -272,8 +272,12 @@ task startStoragesInDocker() {
     outputs.upToDateWhen { false }
 }
 
+task generateScmSourceJson(type: Exec) {
+    commandLine "scm-source", "-f", "scm-source.json"
```

Comment:
Build fails because it can't find 'scm-source' command. What this command is supposed to be?
https://travis-ci.org/zalando/nakadi/jobs/121664971